MinecraftClient/ExitCleanUp.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Runtime.InteropServices;
+
+namespace MinecraftClient
```

Comment:
Since it's specific to Windows, this class should go to `MinecraftClient.WinAPI` folder/namespace.